common.mk

```diff
@@ -41,7 +41,7 @@ LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.Version
 
 # Add supported OS-ARCHITECTURE combinations here
 ENVS ?= linux-amd64 windows-amd64 darwin-amd64
-STANDALONE_PLUGINS ?= login management-cluster package pinniped-auth secret
+STANDALONE_PLUGINS ?= login management-cluster package pinniped-auth secret telemetry
```

Comment:
Considering this is a context-scoped plugin, it should be added to the next array `CONTEXTAWARE_PLUGINS`.